docs/getting-started/dashboard_tutorial.md

```diff
@@ -0,0 +1,1788 @@
+```python exec
+import reflex as rx
+from pcweb.pages import docs
+```
+
+# Tutorial: Data Dashboard
+
+During this tutorial you will build a small data dashboard, where you can input data and it will be rendered in table and a graph. This tutorial does not assume any existing Reflex knowledge, but we do recommend checking out the quick [Basics Guide]({docs.getting_started.basics.path}) first. 
+
+The techniques you’ll learn in the tutorial are fundamental to building any Reflex app, and fully understanding it will give you a deep understanding of Reflex.
+
+
+This tutorial is divided into several sections:
+
+- Setup for the tutorial will give you a starting point to follow the tutorial.
+- Overview will teach you the fundamentals of Reflex: components and props.
```

Comment:
ehh personal preference but I would maybe format these summary bullet points a little differently. something like

- **Setup for the Tutorial**: A starting point to follow the tutorial
- **Overview**: The fundamentals of Reflex UI (components and props)
- **Showing Dynamic Data**: How to use state to render dynamic data. 

I dunno, something that clearly shows that these are subsection headings that you're mentioning, and separating out the description maybe.

Reply:
fixed